main.py

```diff
@@ -875,6 +875,8 @@ def ui_access_wrapper(self, *args, **kwargs):
     get_redirect_route(
         r'%s' % feconf.BLOG_DASHBOARD_DATA_URL,
         blog_dashboard.BlogDashboardDataHandler),
+    get_redirect_route(
+        r'%s' % feconf.BLOG_DASHBOARD_URL, blog_dashboard.BlogDashboardPage),
```

Comment:
Are we going to allow users to access this page in production once this PR is release? [Just to confirm before we add this URL here!]

Reply:
User can access this only after user is assigned role of blog admin or blog post editor, currently no one will be assigned these roles. So, it should be a problem. As such I will need to have all the PRs atleast on test server to check a job that will change blog post data while migrating jobs from medium to our website. Both the job and the blog post editor will be required.